screen/accounts/src/main/java/com/ivy/accounts/AccountsTab.kt

```diff
@@ -320,42 +327,79 @@ private fun AccountHeader(
 @Composable
 private fun PreviewAccountsTab() {
     IvyWalletPreview {
+        val acc1 = com.ivy.data.model.Account(
```

Comment:
The Compose view state must not use domain model objects because #2965 but that can be a follow-up PR